spec/resources/simple_site/hit_counter_page.rb

```diff
@@ -29,6 +29,6 @@
 
 class HitCounterPage < Gamera::Page
   def initialize
-    super('/hit_counter', %r{\/hit_counter})
```

Comment:
You shouldn't need to escape the '/' when using %r quoting, e.g. `%r{/hit_counter}` would work